scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+# fail if some commands fails
+set -e
+# show commands
+set -x
+
+ARCH=$(uname -m)
+export CI="openshift"
+if [ "${ARCH}" == "s390x" ]; then
+    make configure-installer-tests-cluster-s390x
+elif [ "${ARCH}" == "ppc64le" ]; then
+    make configure-installer-tests-cluster-ppc64le
+else
+    make configure-installer-tests-cluster
+fi
+make bin
+mkdir -p $GOPATH/bin
+go get -u github.com/onsi/ginkgo/ginkgo
+export PATH="$PATH:$(pwd):$GOPATH/bin"
```

Comment:
```export PATH="$PATH:$(pwd):$GOPATH/bin"``` -> ```export PATH="$PATH:$(pwd)"```

Adding pwd in the path will help us to detect the `kam` bit

Reply:
${pwd} is already added there ^^^, right?

Reply:
yes